bokeh/core/property/serialized.py

```diff
@@ -0,0 +1,76 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2022, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+""" Provide ``NotSerialized`` property. """
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+from typing import Any, TypeVar
+
+# Bokeh imports
+from ._sphinx import property_link, register_type_link, type_link
+from .bases import (
+    Init,
+    Property,
+    SingleParameterizedProperty,
+    TypeOrInst,
+)
+from .singletons import Intrinsic
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    "NotSerialized",
+)
+
+T = TypeVar("T")
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+class NotSerialized(SingleParameterizedProperty[T]):
+    """
+    A property accepting a value of some other type while having undefined default.
+
+    .. deprecated:: 3.0.0
+
+        Use ``bokeh.core.property.required.Required`` instead.
+    """
+
+    _serialized = False
+
+    def __init__(self, type_param: TypeOrInst[Property[T]], *, default: Init[T] = Intrinsic, help: str | None = None) -> None:
+        super().__init__(type_param, default=default, help=help)
```

Comment:
This should actually call `deprecated`

Reply:
This is still a feature, which actually current test failures indicate (I removed one too many `serialized`).

Reply:
Sorry, I don't follow. The comment indicates it is deprecated, in which case a call to `deprecated` is advised to let users know that fact at runtime.

Reply:
I didn't see enough context in to top-level thread, so I missed the deprecation. I copied this from `NonNullable. I will remove the deprecation.